packages/components/src/draggable/README.md

```diff
@@ -49,10 +51,54 @@ const MyDraggable = () => (
 					elementId="draggable-panel"
 					transferData={ { } }
 				>
-					<Dashicon icon="move" />
+				{
+					( { onDraggableStart, onDraggableEnd } ) => (
+						<Dashicon
+							icon="move"
+							onDragStart={ onDraggableStart }
+							onDragEnd={ onDraggableEnd }
+							draggable
```

Comment:
FYI @nosolosw: the `draggable` attribute is enumerated, so shorthanding it like you have here is actually not valid.

See here: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/draggable

cc @youknowriad

Reply:
Isn't this something already taken care of by React? we should fix if not :)

Reply:
@dsifford you're right in that `draggable` is enumerated. We could probably the example (and [code](https://github.com/WordPress/gutenberg/blob/7c1811268d47181b5a4d77497706ddeb948f6cd1/packages/block-editor/src/components/block-mover/drag-handle.js#L32)) to remain as true to the HTML standard as possible. However, note that React does the [conversion](https://github.com/facebook/react/blob/d38cfd452f014167d223c73091d4a148a9f893a5/packages/react-dom/src/shared/DOMProperty.js#L252) for us.